### DISCUSSION TITLE
Cameras being turned on and off

### DISCUSSION THREAD

**Discussion by geddyleefan** [mrlt8/docker-wyze-bridge#1150] [Q&A]
I have two V3 Pro cameras and they are getting powered off every time the bridge makes a connection:
[WyzeBridge] [CONTROL] ☁️ Sending power_off to living-room-cam via Wyze API
I just got a new V4 camera, and it's doing the reverse:
[WyzeBridge] [CONTROL] ☁️ Sending power_on to office-cam-4 via Wyze API

I can't think of any changes I could have made that would have caused this, how can I get the bridge to stop doing that? It's been working just fine since I installed Home Assistant maybe six months ago. I tried uninstalling it and installing again, but that didn't make any difference.

**Comment by piersm2** [comment]
I am running into a bunch of connect/disconnect instances in my logs as well.
